airbyte-cdk/java/airbyte-cdk/dependencies/build.gradle

```diff
@@ -1,13 +1,22 @@
 import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
```

Comment:
This `dependencies` module was built from all the `airbyte-*` and `*-oss` modules, which are left-overs from the times of the monorepo. I ruled out importing things like airbyte-commons from the platform because of the considerable amount of dependencies they pull there. To be fair, dependencies are a lot less of a concern in that repo than they are here. Still, this code changes little enough that IMO it's safe to maintain two distinct copies of it.